### DISCUSSION TITLE
**in session** [Exercise-6]: Write in your own words about bind variables and conditional statements.

### DISCUSSION THREAD

**Discussion by acrlakshman** [exaslate-courses/oracle-sql-plsql-b3#7] [General]
Write about bind variables and conditional statements.

**Comment by YaminiYadavv** [comment]
We discussed about bind variables and how they work in PL/ SQL part. We need to have : symbol before writing the code of bind variable.
We can assign the values in that code and the bind variables also allows to write the code once and execute it multiple times with different data values.

**Comment by cm2024jsp** [comment]
A bind variable is utilized to link or serve as a bridge between sql plus and pl sql, and we must insert this phrase before any bind variables. It facilitates the conversion of variables from sql plus to pl sql.

**Comment by srideviadapa0511** [comment]
A  bind variable is used to connect or act as bridge between sql plus ans pl sql .we need to put : before any bind variable . It helps in extracting variables from sql plus to pl sql .

**Comment by Tharun1596** [comment]
bind variable acts as a communication between the sql and pl/sql
we need to use colon before code = :v_code
